### PR TITLE
OEL-306: Provide default content feature.

### PR DESCRIPTION
**Pull request by escuriola**
### Description

Added required features to allow to store default content for showcase library.

### Change log

- Added: oe_showcase_default_content submodule
- Changed:
- Deprecated:
- Removed: 
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_showcase_default_content/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# OpenEuropa Showcase default content
+
+Provide default content to oe_showcase pages
```
Comment (GilNovacomm) [inline modules/oe_showcase_default_content/README.md:3]: The rationale here is that in the context of oe_showcase this module allows other modules to provide default content not oe_showcase_default_content directly.
So maybe update the description to reflect this idea.

**Review by GilNovacomm — COMMENTED** [review]
Consider rewriting the README file description.
This module doesn't directly provides default content, in the context of oe_showcase it allows other modules to provide default content at installation.

File: modules/oe_showcase_default_content/README.md (hunk 1 above)
Comment (escuriola) [reply modules/oe_showcase_default_content/README.md:3]: Not exactly. All the content will be stored in this module, as explained in the ticket.

Comment (GilNovacomm) [reply modules/oe_showcase_default_content/README.md:3]: Ok I've confirmed with Andras that this module will also provide default content if this content is reused by other features so the description also applies.

**Review by GilNovacomm — DISMISSED** [review]

**Review by drishu — APPROVED** [review]

**Review by GilNovacomm — APPROVED** [review]

**Pull request merged**